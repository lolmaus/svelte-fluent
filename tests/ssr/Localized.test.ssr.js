import Component from '../../stories/localized/Variables.svelte'

it('renders correctly', () => {
  const result = Component.render({ unreadEmails: 42 })
  expect(result).toMatchSnapshot()
})
