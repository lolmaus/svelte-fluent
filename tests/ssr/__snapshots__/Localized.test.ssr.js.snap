// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "css": Object {
    "code": "",
    "map": null,
  },
  "head": "",
  "html": "
	You have ⁨42⁩ unread emails.
",
}
`;
